.changeset/tidy-dogs-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Adds helpful message for when the API Route export is not uppercase.
```

Comment:
```suggestion
Adds a helpful warning message for when an exported API Route is not uppercase.
```
Possible rewording for the changeset, I hope it helps :)

Reply:
Much better, thanks!